src/main/java/org/apache/commons/text/names/HumanNameParser.java

```diff
@@ -98,8 +98,9 @@
  *
  * <p>This class is immutable.</p>
  */
-public final class HumanNameParser {
+public class HumanNameParser {
```

Comment:
I'm ok reverting this change and starting another ticket or discussion else where. I removed `final` because I am mocking this class in my project's unit tests.